docs/guides/entry_points.md

```diff
@@ -0,0 +1,90 @@
+
+### First example: slow recommendation below the fold
+
+Let’s say you want to offer visitors **targeted recommendations below the fold of your home** page. You currently have simple recommendation in a carousel. Those are fast to load, but not well targetted. The improved, more complicated recommendation system leverages the information you have about a visitor (likely: IP, time of day, day of week, browser type). However, it takes almost a second to run, a very long time for a web server. Usually, visitors only start scroll after reading what is above the fold, so you typically have the time to generate those before the visitor scrolls and the recommendation carrousel enters the viewport. However, only a third of visitors scroll far enough to see the recommendations.
+
+If you want to A/B test the impact of introducing that new recommendation, should you wait for visitors to scroll to the carrousel? That would be a sub-optimal experience: an empty carrousel would appear, then a very long second later, some suggestions pop in. Many visitors would have scrolled futher down at this point. Unless the computation is very expensive, it’s more likely that you want to trigger it for all visitors, so that it’s ready when they scroll it into view. That means that two thirds of the time, a visitors will not see the results.
+
+When testing, should you assign all users? You need to decide whether to trigger the computation (Treatment) or not (Control). That assignment has to happen before you trigger the computation, therefore as soon as the visitor starts loading the home-page. You would need to assign a visit to either Control (older, fast recommendation) or Treatment (newer, better, slower recommendation) as soon as they start requesting the page.
+
+There’s one concern though: while all visitors will be assigned in the experiment, only the visitors who see the recommendation carousel are exposed to a different experience. (We’ll ignore the impact of triggering an expensive computation for now.) Therefore, the two thirds of visitors who were assigned but never scrolled down the homepage should not be included in that experiment.
+
+For cases like that, we let you define an **Entry Point**: what event needs to happen for visitors to be exposed to a different experience, and considered enrolled to the experiment. It remains up to you to decide if this should be when the carousel enters the viewport, is fully or partially visible; it’s also up to your front-end developpers to trigger and log that event. But once that information is in your data warehouse, then you can use it to filter out which users participate in the experiment.
+
+::: note
+If you define an Entry point, all the time-limted metrics (“Conversion 7 days after assignment”) are based on the timestamp of the Entry point, not the assignment. 
```

Comment:
I think this language is a little confusing. My understanding was that entry points were a way to specify that the assignment happens based on an entry point, so the timestamp of the entry point is the same as the timestamp of the assignment. Is that incorrect?